tasks/bug_report_template.rb

```diff
@@ -12,12 +12,20 @@
   end
 
   # Change Rails version if necessary.
-  gem "rails", "6.1.4"
+  gem "rails", "~> 7.0.0"
 
   gem "sprockets", "3.7.2"
   gem "sassc-rails", "2.1.2"
   gem "sqlite3", "1.4.2", platform: :mri
-  gem "activerecord-jdbcsqlite3-adapter", "61.0", platform: :jruby
+  gem "activerecord-jdbcsqlite3-adapter", "70.0", platform: :jruby
+
+  # Prevents an issue with inline bundle when default gems have been
+  # already activated. May not be needed in newer versions of Ruby / bundle
+  # Ref: rubygems/rubygems#6386
+  if ENV["CI"]
+    gem "net-protocol", "0.1.1", platform: :jruby
+    gem "timeout", "0.3.0", platform: :jruby
+  end
```

Comment:
Thank you! 🙌🏻 I could not figure this out for the life of me. The script was moved to the lint.yml workflow for now but it's commented out. Please uncomment it and let's see if its running here now.

Reply:
Still failing. 😔 Not sure what it could be but appreciate you looking into it. If you are able to figure it out, would love to know what the issue is. I can accept the other changes in the meantime. Whether you prefer to resolve and do all at one, or separate the changes, is fine with me. Thank you.

Reply:
>  would love to know what the issue is. 

Now it is MRI Ruby that has activated still different versions. I think this may be fixed in future releases of the offending dependencies. I will do some more tests in my fork and then provide a solution